tools/ci_build/github/azure-pipelines/packaging-pipeline-stable.yml

```diff
@@ -0,0 +1,45 @@
+stages:
+- stage: Python_Packaging
+
+  jobs:
+    - job: Linux_py_GPU_Wheels
+      timeoutInMinutes: 180
+      workspace:
+        clean: all
+      pool: Onnxruntime-Linux-GPU
+      strategy:
+        matrix:
+          Python38 Torch181 onxruntime181 Cuda10.2:
+            PythonVersion: '3.8'
+            PublicDockerFile: 'docker/Dockerfile.ort-torch181-onnxruntime181-cu102-cudnn7-devel-ubuntu18.04'
+          Python38 Torch181 onxruntime181 Cuda11.1:
+            PythonVersion: '3.8'
+            PublicDockerFile: 'docker/Dockerfile.ort-torch181-onnxruntime181-cu111-cudnn8-devel-ubuntu18.04'
+          Python38 Torch190 onxruntime181 Cuda10.2:
+            PythonVersion: '3.8'
+            PublicDockerFile: 'docker/Dockerfile.ort-torch190-onnxruntime181-cu102-cudnn7-devel-ubuntu18.04'
+          Python38 Torch190 onxruntime181 Cuda11.1:
+            PythonVersion: '3.8'
+            PublicDockerFile: 'docker/Dockerfile.ort-torch190-onnxruntime181-cu111-cudnn8-devel-ubuntu18.04'
+            UploadWheel: 'yes'
+          Python38 Default Torch190 onxruntime181 Cuda11.1:
```

Comment:
there was a typo in the original yml. it should be Cuda10.2 instead of Cuda11.1 for the matrix label.